script.js

```diff
@@ -53,25 +83,38 @@ const form = {
     rows: [
         {
             view: "form",
-            width: 250,
+            id: "form",
+            width: 350,
             elements: [
                 {
                     rows: [
                         { template: "EDIT FILMS", type: "section" },
-                        { view: "text", label: "Title" },
-                        { view: "text", label: "Year" },
-                        { view: "text", label: "Rating" },
-                        { view: "text", label: "Votes" },
+                        { view: "text", label: "Title", name: "title", invalidMessage: "Title can not be empty" },
+                        { view: "text", label: "Year", name: "year", invalidMessage: "Year should be between 1970 and current" },
+                        { view: "text", label: "Rating", name: "rating", invalidMessage: "Rating cannot be empty or 0" },
```

Comment:
here you can use 
**type: "number"**
to make it enter only numbers

Reply:
Didn’t know about this type, thanks, will use it.